src/redisearch_rs/inverted_index_bencher/src/benchers/numeric.rs

```diff
@@ -284,7 +284,7 @@ fn numeric_c_decode<M: Measurement>(group: &mut BenchmarkGroup<'_, M>, input: &B
         ),
         |b| {
             for (_, _, buffer) in values {
-                b.iter_batched(
+                b.iter_batched_ref(
```

Comment:
What's the rationale for having the `for` loop outside the `iter_batched_ref` here but not for the encoder benches?